vpp-manager/images/ubuntu-build/Dockerfile

```diff
@@ -27,7 +27,13 @@ RUN apt-get update \
     libssl-dev \
     clang clang-format-11 \
 	libffi7 \
-	enchant-2
+	enchant-2 \
+	libffi7 \
+	libpam-cap tzdata libmaxminddb0  libbcg729-0 liblua5.2-0 libnl-genl-3-200 libsmi2ldbl libspandsp2 \
+	libspeexdsp1 libssh-gcrypt-4 libwireshark-data libc-ares2 libsbc1 libsnappy1v5 libwsutil13 \
+	libwiretap12  libwireshark15 libxslt1.1 python-babel-localedata python3-tz python3-babel \
+	python3-soupsieve python3-bs4 python3-webencodings python3-html5lib  python3-markupsafe python3-jinja2 \
+	python3-lxml  wireshark-common  tshark gcovr
```

Comment:
Are you sure we do need all these dependencies ? It seems like only `tshark` was added between the two 😮 
````bash
git diff 7419bede7ad73544338fd4363da833b2d5fc89a5..2ae8f79b5087c8da7d30283693f22ca710a60347 Makefile
````
gives :
````diff
diff --git a/Makefile b/Makefile
index 98d1d416d..ed7d5b2e5 100644
--- a/Makefile
+++ b/Makefile
@@ -81,6 +81,7 @@ DEB_DEPENDS += iperf3 # for 'make test TEST=vcl'
 DEB_DEPENDS += nasm
 DEB_DEPENDS += iperf ethtool  # for 'make test TEST=vm_vpp_interfaces'
 DEB_DEPENDS += libpcap-dev
+DEB_DEPENDS += tshark

 LIBFFI=libffi6 # works on all but 20.04 and debian-testing

@@ -197,6 +198,7 @@ ifneq ($(wildcard $(STARTUP_DIR)/startup.conf),)
 endif

 ifeq ($(findstring y,$(UNATTENDED)),y)
+DEBIAN_FRONTEND=noninteractive
 CONFIRM=-y
 FORCE=--allow-downgrades --allow-remove-essential --allow-change-held-packages
 endif
````

Reply:
you're right, I just blindly copied all dependencies needed for the 3.26 vpp rebase